actors/builtin/miner/miner_test.go

```diff
@@ -141,18 +141,19 @@ func TestControlAddresses(t *testing.T) {
 		assert.Equal(t, actor.worker, w)
 	})
 
-	// TODO: test changing worker (with delay), changing peer id
-	// https://github.com/filecoin-project/specs-actors/issues/479
 }
 
 // Test for sector precommitment and proving.
 func TestCommitments(t *testing.T) {
 	periodOffset := abi.ChainEpoch(100)
 
 	// TODO more tests
+	// https://github.com/filecoin-project/specs-actors/issues/936
 	// - Concurrent attempts to upgrade the same CC sector (one should succeed)
-	// - Insufficient funds for pre-commit, for prove-commit
+	// - Insufficient funds for prove-commit (charged in power actor -- scenario testing candidate)
```

Comment:
Since these are now covered in an issue (Thanks!), do they still need to be documented as comments? My concern is that this becomes a redundant and eventually inconsistent work queue.

Reply:
Yeah great point, I will delete